src/routes/Settings/SettingsIndexView.js

```diff
@@ -1,7 +1,164 @@
 // @flow
 
-function SettingsIndexView() {
-  return null
+import React, { PureComponent } from 'react'
+
+import { JText } from 'components/base'
+import { SettingsGrid, SettingsGridCard } from 'components'
+
+import { divideThousands } from 'utils/numbers'
+import { formatBoolean, formatCurrency, formatLanguage } from './utils'
+
+type Props = {|
+  ...SettingsState,
+  networkName: ?string,
+  wallet: ?Wallet,
+|}
+
+class SettingsIndexView extends PureComponent<Props, *> {
+  render() {
+    const {
+      localCurrencyCode,
+      defaultGasPrice,
+      systemLanguageCode,
+      hasPinCode,
+      networkName,
+      wallet,
+    } = this.props
+
+    if (!wallet) {
+      return null
+    }
+
+    const { name: walletName, customType, isReadOnly, mnemonicOptions } = wallet
+    const isFullMnemonic = customType === 'mnemonic' && !isReadOnly
+    const derivationPath = mnemonicOptions ? mnemonicOptions.derivationPath : null
```

Comment:
> customType === 'mnemonic' && !isReadOnly

better to use brackets when condition contains more than one logical operator. To make it easy to read

Reply:
It's very simple expression, I think it's not necessary.